osversion/platform_compat_windows.go

```diff
@@ -0,0 +1,70 @@
+package osversion
+
+import (
+	"strconv"
+	"strings"
+)
+
+// List of stable ABI compliant
+// ltsc releases
+// Note: List must be sorted
+var compatLtscReleases = []uint{
```

Comment:
Standard Go formatting is to capitalize acronyms in identifiers, so would be `compatLTSCReleases`.